src/components/ContactCard/ContactForm.jsx

```diff
@@ -6,6 +6,7 @@ import ContactFormField from "./ContactFormField";
 import ContactFieldInput from "./ContactFieldInput";
```

Comment:
A little extra with this PR: we want to render the buttons for the creation form… elsewhere. Generally in a fixed modal footer.

I've used portals to make that possible, and successfully used it in the intent. However, when using this with the `ContactFormModal`, we run into a preact-bug when closing the Modal, and I'm too lazy to report it.